client/app/components/category_list.vue

```diff
@@ -0,0 +1,6 @@
+<template lang="pug">
+    main
+        div(role='contentinfo')
```

Comment:
The `[role=contentinfo]` could be put on the `main` tag, this div isn't usefull (except if it's here for layouts concerns)